PhysicsTools/PatAlgos/python/slimming/applySubstructure_cff.py

```diff
@@ -131,7 +76,10 @@ def applySubstructure( process, postfix="" ) :
         e.toModify(getattr(process,'nb1AK8PuppiSoftDrop'+postfix), cuts = ['pt > 999999', 'pt > 999999', 'pt > 999999'] )
         e.toModify(getattr(process,'nb2AK8PuppiSoftDrop'+postfix), cuts = ['pt > 999999', 'pt > 999999', 'pt > 999999'] )
 
-
+    from RecoJets.JetProducers.nJettinessAdder_cfi import Njettiness
+    addToProcessAndTask('NjettinessAK8Subjets'+postfix, Njettiness.clone(), process, task)
+    getattr(process,"NjettinessAK8Subjets"+postfix).src = cms.InputTag("ak8PFJetsPuppiSoftDrop"+postfix, "SubJets")
+    getattr(process,"NjettinessAK8Subjets"+postfix).R0 = 0.8
```

Comment:
this parameter is already set to 0.8
https://github.com/cms-sw/cmssw/blob/783cecb12c181a0b36f30a144fe1d48bd0dd0008/RecoJets/JetProducers/python/nJettinessAdder_cfi.py#L9
what is the reason for the explicit assignment here?

Reply:
Just for backwards "compatibility". I guess it can go if you want.